### ISSUE TITLE
FCI Floating point exception 2.5.0

### ISSUE THREAD

**Issue by FedeRoxx** [pyscf/pyscf#2184]
I am trying to get FCI energies but the same input that runs fine with version 2.4.0 of pyscf, crashes with version 2.5.0.

Here's the input:

```
import pyscf

mol = pyscf.M(atom="He 0.0 0.0 0.0; H -0.409760 1.631325 0.0; H 0.7 0.0 0.0", basis="aug-ccPVDZ")

mf = pyscf.scf.RHF(mol)
mf.run()

cisolver = pyscf.fci.FCI(mf)
cisolver.nroots = 5
cisolver.kernel()
```
Running with pyscf version 2.4.0 everything works fine.
If I try to run the same input with version 2.5.0, the FCI crashes and the output I get is
```
converged SCF energy = -3.62468169582011
Floating point exception (core dumped)
```

Changing a lot the geometry or basis set avoids the problem.

**Comment by sunqm** [comment]
Related to #2105. Fixed by PR #2106

**Comment by FedeRoxx** [comment]
Hi @verena-neufeld, yes it is solved now. Thank you!
